spec/requests/api/v1/category_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Api::V1::Categories', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/api/v1/category/index'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns http success' do
+      get '/api/v1/category/show'
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET /create' do
+    it 'returns http success' do
+      get '/api/v1/category/create'
+      expect(response).to have_http_status(:success)
```

Comment:
You have to leave this file without any example, because there's not any action in the controller.